### PR TITLE
🔀🔖 4.5.1

### PR DESCRIPTION
**Pull request by jxn-30**
* 🐛 [emv] fix missing extras text
* 🐛 [redesign/verband/mitglieder] fix buttons not shown when alliance funds disabled
* 🐛 [sap] fix keypress enter on shared missions
* 🚸 [heatmap] improve size of settings window
* 🚸 [heatmap] settings window can be moved
* ✨ [ce] add option to hide sales
* ✨ [em] settings: button "use current position" added to map.
* ✨ [heatmap] live preview
* ✨ [sap] new features:
    * new variables:
        * `{{name}}`: Name of the mission
        * `{{today}}`: current local date (day and month)
    * live edit a text before sharing

### CODE REVIEW

File: src/modules/heatmap/i18n/en_AU.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
 		"vehicles": "vehicle types"
 	},
 	"intensity": "intensity boost",
+	"livePreview": "Live-Preview (auto-save)",
```
Comment (Cr4zyc4k3) [inline src/modules/heatmap/i18n/en_AU.json:9]: "Autosave" is a better translation

File: src/modules/heatmap/i18n/nl_NL.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
 		"vehicles": "Voertuig types"
 	},
 	"intensity": "Intensiteit",
+	"livePreview": "Live-Preview (auto-save)",
```
Comment (Cr4zyc4k3) [inline src/modules/heatmap/i18n/nl_NL.json:9]: "automatisch opslaan"

File: src/modules/shareAlliancePost/docs/de_DE.md [hunk 3]
```diff
@@ -30,10 +30,18 @@ Um auch die Postleitzahl wegzulassen, verwende <code><span>{{</span>cityWithoutZ
...
+
+<code><span>{{</span>name<span>}}</span></code> fügt den Namen des Einsatzes ein.
+
 ### Längste Anfahrt
 
 <code><span>{{</span>longestDrive<span>}}</span></code> fügt ein, wie lange alle aktuell ausgewählten Fahrzeuge benötigen, um anzufahren. **Wichtig**: Diese Variable ist in Rückmeldungen aus der Einsatzliste heraus natürlich nicht verfügbar!
 
+### Datum
+
+<code><span>{{</span>today<span>}}</span></code> fügt das heutige Datum (Tag und Monat) ein. z.B. `13.12`
```
Comment (Cr4zyc4k3) [inline src/modules/shareAlliancePost/docs/de_DE.md:43]: 1312 = ACAB may change?

**Review by Cr4zyc4k3 — APPROVED** [review]

File: src/modules/heatmap/i18n/nl_NL.json (hunk 2 above)
Comment (JRH-1997) [reply src/modules/heatmap/i18n/nl_NL.json:9]: `Live voorvertoning (automatisch opslaan)`

**Review by Suchty112 — APPROVED** [review]

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**